File: src/NetWorthCalculator.Web.Service/Controllers/BalanceSheet/Dto/AssetDto.cs
using NetWorthCalculator.Entities.Enums;

namespace NetWorthCalculator.Web.Service.Controllers.BalanceSheet.Dto
{
	public class AssetDto
	{
		public int Id { get; set; }

		public decimal Amount { get; set; }

		public string Description { get; set; }

		public AssetGroup Group { get; set; }

		public int Order { get; set; }
	}
}

File: src/NetWorthCalculator.Web.ReactJs/src/Services/BalanceSheet/Dto/AssetDto.ts
import { serializable } from "serializr";
import { AssetGroup } from "../../../Models/Asset/AssetGroup";

export default class AssetDto {
	@serializable id!: number;

	@serializable amount!: number;

	@serializable group!: AssetGroup;

	@serializable description!: string;

	@serializable order!: number;
}